x/deployment/keeper/keeper.go

```diff
@@ -147,6 +147,24 @@ func (k Keeper) WithDeployments(ctx sdk.Context, fn func(types.Deployment) bool)
 	}
 }
 
+// WithDeploymentsActive filters to only those with State: Active
+func (k Keeper) WithDeploymentsActive(ctx sdk.Context, fn func(types.Deployment) bool) {
```

Comment:
`WithActiveDeployments()` rolls off the tongue better.  Or make the state a parameter `WithDeploymentsInState(ctx,state,fn)`